### PR TITLE
Update some descriptions.

### PR DESCRIPTION
**Pull request by xingwu1**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by xingwu1** [comment]
@matthchr @darylmsft Could you review my PR?

**Comment by veronicagg** [comment]
@xingwu1 if you'd like to view how the updates show in the documentation you can access [this preview link](https://review.docs.microsoft.com/en-us/rest/api/azure-rest-preview/?view=azure-rest-preview&branch=openapiHub_production_01810dd4dd7a) available from the Documentation section at your [work branch page in OpenAPIHub](https://portal.azure-devex-tools.com/app/branch/xingwu1/azure-rest-api-specs/master).
If you're good with that, let me know and I'll merge this change.

File: specification/batch/data-plane/Microsoft.Batch/stable/2018-12-01.8.0/BatchService.json [hunk 1]
```diff
@@ -13710,7 +13712,7 @@
       },
       "required": [ "coordinationCommandLine" ],
       "title": "Settings which specify how to run a multi-instance task.",
-      "description": "Multi-instance tasks are commonly used to support MPI tasks."
+      "description": "Multi-instance tasks are commonly used to support MPI tasks. In MPI scenario, if any of the subtasks fail, for example, exiting with a non-zero return code, the entire multi-instance task fails. The multi-instance task is then terminated and retried, up to its retry limit."
```
Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/stable/2018-12-01.8.0/BatchService.json]: Maybe:
> Multi-instance tasks are commonly used to support MPI tasks. In the MPI case, if any of the subtasks fail (for example due to exiting with a non-zero exit code) the entire multi-instance task fails. The multi-instance task is then terminated and retried, up to its retry limit.

Comment (xingwu1) [reply specification/batch/data-plane/Microsoft.Batch/stable/2018-12-01.8.0/BatchService.json]: Updated.

**Review by veronicagg — APPROVED** [review]

**Pull request merged**